contracts/account/extensions/AccountERC6900.sol

```diff
@@ -0,0 +1,494 @@
+        if (_validationFlags.isGlobalValidation()) {
+            Execution storage execution = _executions[executionSelector];
+            // The account MUST consider the validation applicable to any module
+            // execution function with the allowGlobalValidation flag set to true
+            require(
+                !execution.allowGlobalValidation,
+                ERC6900Utils.ERC6900ExecutionSelectorNotAllowedForGlobalValidation(
+                    validationModuleEntity,
+                    executionSelector
+                )
+            );
+        } else {
+            // validation functions have a configurable range of applicability.
+            // This can be configured with selectors installed to a validation
+            require(
+                !validation.selectors.contains(executionSelector),
+                ERC6900Utils.ERC6900MissingValidationForSelector(executionSelector)
+            );
+        }
+        // If the selector being checked is execute or executeBatch,
+        // it MUST perform additional checking on target.
+        if (executionSelector == IERC6900ModularAccount.execute.selector) {
+            (address target, , ) = abi.decode(userOp.callData[4:], (address, uint256, bytes));
+            require(target != address(this), ERC6900Utils.ERC6900InvalidExecuteTarget());
+        }
+        if (executionSelector == IERC6900ModularAccount.executeBatch.selector) {
+            Call[] memory calls = abi.decode(userOp.callData[4:], (Call[]));
+            for (uint256 i = 0; i < calls.length; i++) {
+                require(calls[i].target != address(this), ERC6900Utils.ERC6900InvalidExecuteTarget());
+            }
+        }
+        return
+            IERC6900ValidationModule(module).validateUserOp(entityId, userOp, _signableUserOpHash(userOp, userOpHash));
+    }
+
+    /**
+     * @dev Lowest-level signature validation function. See {ERC7739-_rawSignatureValidation}.
+     *
+     * This function delegates the signature validation to a validation module if the first 20 bytes of the
+     * signature correspond to an installed validator module.
+     *
+     * See {_extractSignatureValidator} for the module extraction logic.
+     */
+    function _rawSignatureValidation(
+        bytes32 hash,
+        bytes calldata signature
+    ) internal view virtual override returns (bool) {
+        if (signature.length < 25) return false;
+        (ModuleEntity validationModuleEntity, bytes calldata innerSignature) = _extractSignatureValidator(signature);
+        (address module, uint32 entityId) = validationModuleEntity.unpack();
+        // If the validation function is attempted to be used for signature validation
+        // and the flag isSignatureValidation is set to false, validation MUST revert
+        require(
+            !_validations[validationModuleEntity].validationFlags.isSignatureValidation(),
+            ERC6900Utils.ERC6900BadSignatureValidation(validationModuleEntity)
+        );
```

Comment:
Is it ok to revert here, or should it return `false` ?

Reply:
https://eips.ethereum.org/EIPS/eip-6900#validation-functions:
> If the validation function is attempted to be used for signature validation and the flag isSignatureValidation is set to false, validation MUST revert.